src/pages/thanks-page/thanks-card/thanks-card.styles.js

```diff
@@ -14,16 +15,16 @@ export const useStyles = makeStyles(({ palette }) => ({
   imageWrapper: {
     background: `url(${BG}) no-repeat center center`,
     backgroundSize: 'cover',
-    height: '65%',
-    width: '50vh',
+    height: '600px',
```

Comment:
Fixed height and width have significant risks for responsiveness.

Reply:
Yeah, but, then the picture changes it's size relative to the screen, and the block with information is fixed and it does not look very good, also, image has display: none when the width is less than 959. 
So, should I change it?